Gemfile

```diff
@@ -120,6 +120,8 @@ gem 'flipper-ui'
 
 gem "view_component", require: "view_component/engine"
 
+gem 'webpacker', '6.0.0.beta.7'
```

Comment:
Maybe v5.3.0 is more stable?

Otherwise, if you want v6, I think you can leave this empty or at least a bit more generic like ~> 6.0?

Reply:
5.3.0 is probably more stable, but, as far as i understood, I need the 6.x

If I write `~> 6.0` I get the `6.0.0.pre.2` which is older than the beta (it took me 2 hours of debugging...) (and I need the last version ... which contains less bugs)

Reply:
ok, we will probably get a stable v6 version before we get this PR to prod 👍